File: CodeChef/Rain in Chefland DR: 328/RAINFALL11.java
import java.util.*;
import java.lang.*;
import java.io.*;

/* Name of the class has to be "Main" only if the class is public. */
class Codechef
{
	public static void main (String[] args) throws java.lang.Exception
	{
		Scanner in = new Scanner(System.in);
		
		int t = in.nextInt();
		
		for(int i = 0; i < t; i++){
		    int x = in.nextInt();
		    
		    if( x < 3){
		        System.out.println("LIGHT");
		    }
		    else if( x >= 7){
		        System.out.println("HEAVY");
		    }
		    else{
		        System.out.println("MODERATE");
		    }
		}
	}
}

File: CodeChef/Rain in Chefland DR: 328/RAINFALL11.cpp
#include <iostream>
using namespace std;

int main() {
    
    int t,x;
    
    cin >> t;
    
    while(t--){
        cin >> x;
        
        if( x < 3){
            cout << "LIGHT";
        }
        else if( x >= 7){
            cout << "HEAVY";
            
        }
        else{
            cout << "MODERATE";
        }
        cout << endl;
        
    }
    
    return 0;
}
